zilencer/management/commands/populate_db.py

```diff
@@ -546,18 +547,18 @@ def handle(self, **options: Any) -> None:
                     "sales": {"description": "For sales discussion"}
                 }
 
-                # Calculate the maximum number of digits in any extra stream's
-                # number, since a stream with name "Extra Stream 3" could show
-                # up after "Extra Stream 29". (Used later to pad numbers with
-                # 0s).
-                maximum_digits = len(str(options['extra_streams'] - 1))
+                # For testing really large batches:
+                # Create extra streams with semi realistic names to make search
+                # functions somewhat realistic.
+                streamname = ['24hrs', 'android', 'automated help', 'backend dev', 'bot',
+                              'checkins', 'code review', 'commits', 'Current User status', 'desktop',
+                              'development help', 'desktop', 'discussions', 'Django guide', 'GCI',
+                              'git test', 'GSoC archive', 'issues', 'live', 'markdown', 'mobile',
+                              'new members', 'provision', 'terminal testing', 'user guide', '测试',
+                              '冰未寒']
```

Comment:
Where did this list of stream names come from?

Reply:
@timabbott I asked on the zulip community if the adding an "extra stream name" list (same as username list we use to create extra users) would be better than using the "Extra Stream 00" format.

Please check https://chat.zulip.org/#narrow/stream/49-development-help/topic/populate_db
for the discussion of the above.